test/bootstrap.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// phpcs:ignoreFile
```

Comment:
This seems a little extreme, what's blowing up in here?

Reply:
> A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line 14 and the first side effect is on line 8.

Essentially it exports a static class for the tests to use but also calls a function within the class outside of it

Given this is a bootstrap file for phpunit, I don't quite know how to solve this without refactoring all the tests